packages/pages/src/common/src/function/internal/validateFunctionModule.ts

```diff
@@ -0,0 +1,36 @@
+import { FunctionModule, FunctionConfig } from "../types.js";
+
+export const validateFunctionDefaultExport = (
+  filepath: string,
+  serverlessFunction: FunctionModule
+) => {
+  if (!serverlessFunction.default) {
+    throw new Error(`${filepath} is missing a default export.`);
+  }
+};
+
+export const validateConfig = (
```

Comment:
We don't need this now right? Or did you mean to validate the Internal type (vs external)? We can validate this for the internal but for external the user isn't defining this anymore.